# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/fake/mythtv/no-ldconfig.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./programs/mythfrontend/mythfrontend.pro.orig	2004-09-14 09:10:44.525994752 +0200
+++ ./programs/mythfrontend/mythfrontend.pro	2004-09-14 09:11:08.855296136 +0200
@@ -20,7 +20,6 @@
 setting.files += recpriorities_settings.xml tv_search.xml tv_lists.xml
 setting.files += library.xml manage_recordings.xml optical_menu.xml tvmenu.xml
 setting.files += tv_settings.xml
-setting.extra = -ldconfig
 
 INSTALLS += setting
 
